slack.opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "slack"
+version: "0.1"
+license: "MIT"
+maintainer: "Ahrefs <github@ahrefs.com>"
+authors: "Ahrefs <github@ahrefs.com>"
+homepage: "https://github.com/ahrefs/slack"
+bug-reports: "https://github.com/ahrefs/slack/issues"
+dev-repo: "git+https://github.com/ahrefs/slack.git"
+
+synopsis: "Slack API implementation."
+description: "OCaml interface for accessing Slack APIs and receiving events."
+
+depends: [
+  "dune"
+  "atdgen" {> "2.0.0"}
+  "atdgen-runtime" {> "2.0.0"}
+  "devkit" {>= "1.4+20200305" }
```

Comment:
We probably want to use the opensource version `devkit.1.20210517`

Reply:
addressed in https://github.com/ahrefs/slack/pull/1/commits/ea3db09df41e4ca00b303dbf4ab00822fc9ab1c9